### PR TITLE
Unignore ESLint check on tests

### PR DESCRIPTION
**Pull request by vicnicius**
## Description of the pull request

Unignore ESLint check on tests.

### CODE REVIEW

File: src/components/Message/MessageSimple.js [hunk 1]
```diff
@@ -214,6 +214,7 @@ class MessageSimple extends PureComponent {
     if (!this.props.message?.deleted_at) {
       document.removeEventListener('click', this._closeDetailedReactions);
       document.removeEventListener('touchend', this._closeDetailedReactions);
+      document.removeEventListener('click', this._hideOptions);
```
[the remark this answers is not in the document]
Comment (ath92) [reply src/components/Message/MessageSimple.js:217]: Other than that it looks good to me :)

[the remark this answers is not in the document]
Comment (vicnicius) [reply src/components/Message/MessageSimple.js:217]: It is a separate change, indeed. I split it to its own commit https://github.com/GetStream/stream-chat-react/pull/288/commits/cbff6c12986247902d2c6880fe0d95aae69a419e. Thanks! 😄

File: src/components/MessageList/__tests__/MessageList.test.js [hunk 2]
```diff
@@ -13,18 +13,17 @@ import {
   generateUser,
   dispatchMessageNewEvent,
   getTestClientWithUser,
-} from 'mock-builders';
+} from '../../../mock-builders';
```
Comment (vishalnarkhede) [inline src/components/MessageList/__tests__/MessageList.test.js:16]: `mock-builders` looks better, no?

Comment (vishalnarkhede) [reply src/components/MessageList/__tests__/MessageList.test.js:16]: we have module resolver for this in jest.config.js

Comment (vishalnarkhede) [reply src/components/MessageList/__tests__/MessageList.test.js:16]: Same for `ChannelList.test.js`

Comment (vicnicius) [reply src/components/MessageList/__tests__/MessageList.test.js:16]: ESLint was complaining about it and I overlooked our jest configuration. I'll adjust ESLint's configuration to fix it properly. Thanks! :D

Comment (vicnicius) [reply src/components/MessageList/__tests__/MessageList.test.js]: Done here: https://github.com/GetStream/stream-chat-react/pull/288/commits/80a0d733ad3b0c3112a851276ed2b7c15f83ff97

**Pull request merged**